dev/com.ibm.ws.request.probe.audit.rest/src/com/ibm/ws/request/probe/audit/rest/service/AuditPERestServiceImpl.java

```diff
@@ -0,0 +1,101 @@
+package com.ibm.ws.request.probe.audit.rest.service;
+
+import java.util.Map;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.security.audit.AuditConstants;
+import com.ibm.ws.security.audit.source.rest.MemberManagementEvent;
+import com.ibm.ws.security.audit.source.rest.RESTAuthorizationEvent;
+import com.ibm.ws.security.audit.source.AuditServiceImpl;
+
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Deactivate;
+
+import com.ibm.wsspi.security.audit.AuditService;
+import com.ibm.wsspi.kernel.service.location.VariableRegistry;
+import com.ibm.wsspi.kernel.service.utils.AtomicServiceReference;
+
+import com.ibm.ws.request.probe.audit.servlet.AuditPERestService;
+
+@Component(service = AuditPERestService.class, configurationPolicy = ConfigurationPolicy.IGNORE,
+           property = "service.vendor=IBM", immediate = true)
+
+public class AuditPERestServiceImpl implements AuditPERestService {
+
+
+	private static final String VARIABLE_REGISTRY_SERVICE = "variableRegistryService";
+	    private static final AtomicServiceReference<VariableRegistry> variableRegistryServiceRef = new AtomicServiceReference<VariableRegistry>(VARIABLE_REGISTRY_SERVICE);
+		private final ConcurrentHashMap<String, List<Map<String, Object>>> handlerEventsMap = new ConcurrentHashMap<String, List<Map<String, Object>>>();
+		private static final TraceComponent tc = Tr.register(AuditPERestServiceImpl.class);
+
+	@Activate
+    protected void activate(ComponentContext cc, Map<String, Object> configuration) {
+        Tr.info(tc, "AUDIT_REST_SERVICE_STARTING");
+        variableRegistryServiceRef.activate(cc);
+
+    }
```

Comment:
```suggestion
    protected void activate(ComponentContext cc) {
    }
```
Maybe just add a debug statement in the activate.  I removed the Tr.info since i didn't see a message file update for a new message.